mqtt/mqtt-broker/src/configuration.rs

```diff
@@ -35,21 +35,17 @@ impl TryFrom<Transport> for TransportBuilder<String> {
             Transport::Tcp { address } => Ok(Self::Tcp(address)),
             Transport::Tls {
                 address,
-                srv_cert_path,
-            } => {
-                load_identity(srv_cert_path.as_path()).map(|identity| Self::Tls(address, identity))
-            }
+                certificate,
+            } => load_identity(certificate.as_path()).map(|identity| Self::Tls(address, identity)),
         }
     }
 }
 
 fn load_identity(path: &Path) -> Result<Identity, Error> {
     info!("Loading identity from {:?}", path);
     let cert_buffer = std::fs::read(&path).context(ErrorKind::LoadIdentity)?;
```

Comment:
Could you please add PathBuf to an ErrorKind
```suggestion
    let cert_buffer = std::fs::read(&path).context(ErrorKind::LoadIdentity(path.to_path_buf())?;
```